Makefile

```diff
@@ -62,6 +62,25 @@ e2etests: ## Run the e2e suite against your local cluster
 		--ginkgo.grace-period=3m \
 		--ginkgo.vv
 
+e2etests-non-flaky: ## Run only non-flaky e2e tests against your local cluster
+	# Notes:
+	# -p: the number of programs, such as build commands or test binaries, that can be run in parallel?
+	# -count 1: prevents caching
+	# -timeout: If a test binary runs longer than TEST_TIMEOUT, panic
+	# -v: verbose output
+	# --ginkgo.label-filter="!flaky": excludes tests labeled as flaky
+	cd test && AZURE_CLUSTER_NAME=${AZURE_CLUSTER_NAME} AZURE_ACR_NAME=${AZURE_ACR_NAME} AZURE_RESOURCE_GROUP=${AZURE_RESOURCE_GROUP} AZURE_SUBSCRIPTION_ID=${AZURE_SUBSCRIPTION_ID} AZURE_LOCATION=${AZURE_LOCATION} go test \
+		-p 1 \
+		-count 1 \
+		-timeout ${TEST_TIMEOUT} \
+		-v \
+		./suites/$(shell echo $(TEST_SUITE) | tr A-Z a-z)/... \
+		--ginkgo.focus="${FOCUS}" \
+		--ginkgo.label-filter="!flaky" \
```

Comment:
Instead of copying, maybe just add --label-filter="{LABEL_FILTER}" capability to e2etest target?